docs/getting_started.md

```diff
@@ -18,9 +18,9 @@ chmod +x /usr/local/bin/argocd
 
 ## 2. Install ArgoCD
 ```
-argocd install
+kubectl apply -f manifests/. --namespace argocd
 ```
-This will create a new namespace, `argocd`, where ArgoCD services and application resources will live.
+This will create a new namespace, `argocd`; where ArgoCD services and application resources will live.
```

Comment:
These instructions still need to change to kubectl apply http://..../install.yaml

Reply:
yeah, was waiting for full update until we had helm discussion but sounds like we want to wait on this, I have updated the docs. thanks @jessesuen